.idea/Fresh.iml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/.meteor/local" />
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```

Comment:
There's lots of these .idea files that come with jetbrain webstorm. Should we add them to .gitignore because we don't need thing (I think). Thoughts @YaoLiu93 @inquisitiv-e ?

Resource: https://stackoverflow.com/questions/43198273/which-files-in-idea-folder-should-be-tracked-by-git

Reply:
Yes please add `.idea` to gitignore

Reply:
![image](https://user-images.githubusercontent.com/17561746/58752056-32c35d80-845d-11e9-83f7-526f9c069675.png)

I deleted the .idea folder entirely and branch ran fine.

Reply:
I ended up deleting the .idea folder